nixnet/database/_lin_sched.py

```diff
@@ -43,45 +45,143 @@ def __repr__(self):
         return '{}(handle={})'.format(type(self).__name__, self._handle)
 
     @property
-    def clst_ref(self):
-        return _props.get_lin_sched_clst_ref(self._handle)
+    def clst(self):
```

Comment:
Do we really need to use names like "clst" and "frm" instead of "cluster" and "frame"? It feels weird to me that we're using different names from the C properties but only going halfway toward making them clear.

Reply:
Something I should have done when we started the project but never did was write up a style guide to explain this.  Hopefully @jashnani  is working on it?

My original intent
- Naming in the bindings maps directly to C (minus PEP8-ifying them)
- The user-facing API reuses the existing C API abbreviations assuming those were chosen for a reason
- Deviate in naming when we are doing something different.  For example, we are no longer returning a cluster reference / handle but instead a cluster object, so the suffix no longer is needed.

Reply:
My first push of this PR did attempt to rename things like frm to frame and clst to cluster, but @epage and @jashnani requested that we not deviate from the C API. This is for the reasons @epage just mentioned. I would definitely like to see this naming revisited in the near future.